### ISSUE TITLE
if my dataset have both long text and very short text that needs to detect,what parameters should i modify?and how?

### ISSUE THREAD

**Issue by yogixRush** [eragonruan/text-detection-ctpn#144]

**Comment by eragonruan** [comment]
@yogixRush I don't think you have to modifty anything, but the results on very short text like single character would be good enough compared to long texts

**Comment by yogixRush** [comment]
@eragonruan  thanks for answering . in my image, there are multiple text areas in one line , but ctpn predicted them as one text area, which is not the same with the ground truth label in my training data , is there any parameter about this?

**Comment by eragonruan** [comment]
@yogixRush you can find all the parameters about text_connector in [lib/text_connector/text_connect_cfg.py](https://github.com/eragonruan/text-detection-ctpn/blob/master/lib/text_connector/text_connect_cfg.py). if you are taiking about different height of texts in ome line, try modify MIN_V_OVERLAPS to 0.8 or 0.9

**Comment by liangshuang1993** [comment]
@eragonruan Hi, you said "the results on very short text like single character would be good enough compared to long texts", but when I tried your model, I found out results on single character is not good enough.  

![ccc0](https://user-images.githubusercontent.com/7533616/42767900-b5b4e3fc-8950-11e8-88a4-7a4146e2bb20.png)


Do you have any suggestions on how to detect pictures like this? Thank you!

**Comment by eragonruan** [comment]
@liangshuang1993 sorry, my mistake. what I mean is the results on single character will not be as good as long texts since ctpn is proposed for line level detection. 
As for your picture, If it is clearly enough and without rotation, maybe you don‘t have to use deep learning method. A simple projection method would be enough.
Or, you can try [EAST](https://github.com/argman/EAST)
